asab/storage/elasticsearch.py

```diff
@@ -50,27 +52,33 @@ def __init__(self, app, service_name, config_section_name='asab:storage'):
 		super().__init__(app, service_name)
 		self.Loop = app.Loop
 
-		self.URL = Config.get(config_section_name, 'elasticsearch_url')
-		parsed_url = urllib.parse.urlparse(self.URL)
-		self.ServerUrls = [
-			urllib.parse.urlunparse((parsed_url.scheme, netloc, parsed_url.path, None, None, None))
-			for netloc in parsed_url.netloc.split(',')
-		]
+		# new configuration format [elasticsearch]
+		url, username, password, api_key = 'url', 'username', 'password', 'api_key'
```

Comment:
Do one `if` branch that reads explicitly the old format, then the `else` branch that reads a new format.